chaos.py

```diff
@@ -0,0 +1,27 @@
+#!/bin/python3
+
+import sys
+
+def minimumBribes(q):
+    # Complete this function
+    bribes = len(q)
+    pos = {1,2}
+    for i in range(n):
+        pos |= {i+3}

```

Comment:
Better to use:
pos.add(i + 3)

This is less readable and technically doing set union, which may have some overhead.